src/components/DiscourseComments.tsx

```diff
@@ -2,12 +2,25 @@ import React, { useState, useEffect } from 'react'
 import { useConnectedGarden } from '@providers/ConnectedGarden'
 import CustomSanitizedHTML from '@/components/SanitizedHTML'
 import env from '@/environment'
```

Comment:
Would like to give a final review once I can access a proposal page with comments.